plugin/src/App/StatusPages/Connected.lua

```diff
@@ -13,6 +13,26 @@ local BorderedContainer = require(Plugin.App.Components.BorderedContainer)
 
 local e = Roact.createElement
 
+local AGE_UNITS = { {31556909, "year"}, {2629743, "month"}, {604800, "week"}, {86400, "day"}, {3600, "hour"}, {60, "minute"}, }
```

Comment:
"last synced one year ago" would be a pretty funny message to happen in practice